vertical-pod-autoscaler/e2e/v1/common.go

```diff
@@ -250,6 +250,17 @@ func InstallVPA(f *framework.Framework, vpa *vpa_types.VerticalPodAutoscaler) {
 	gomega.Expect(err).NotTo(gomega.HaveOccurred(), "unexpected error creating VPA")
 }
 
+// InstallRawVPA installs a VPA object passed in as raw json in the test cluster.
+func InstallRawVPA(f *framework.Framework, obj interface{}) error {
+	vpaClientSet := getVpaClientSet(f)
+	err := vpaClientSet.AutoscalingV1().RESTClient().Post().
```

Comment:
Nit: return vpaClientSet.AutoscalingV1().... ?

Reply:
I'd rather not, this way it is clear that i Do() the request and then return err.Error()

Reply:
Ack